lib/srv/desktop/windows_server.go

```diff
@@ -106,9 +95,7 @@ type WindowsService struct {
 	ca *windows.CertificateStoreClient
 	lc *windows.LDAPClient
 
-	mu              sync.Mutex // mu protects the fields that follow
-	ldapInitialized bool
-	ldapCertRenew   *time.Timer
+	mu sync.Mutex // mu protects the fields that follow
```

Comment:
I don't think you need this at all anymore - it was only to protect `ldapInitialized` and `ldapCertRenew`.

Reply:
This was my biggest question here - it didn't look like it was used for anything else but I wasn't 100% sure. Thanks!